sources/rest_api/__init__.py

```diff
@@ -253,6 +256,18 @@ def create_resources(
             endpoint_resource, {"endpoint", "include_from_parent"}
         )
 
+        def process(
+            resource: DltResource,
+            processing_steps: List[ProcessingSteps] = [],
```

Comment:
Mutable defaults in Python may cause unpredicted behaviour: https://docs.astral.sh/ruff/rules/mutable-argument-default/
Since we always call `processing_steps` the default is not needed here.

Reply:
Amazing behaviour :) Fixed